orttraining/orttraining/eager/opgen/opgen/atenops.py

```diff
@@ -93,19 +93,6 @@ def __init__(self, dY, X):
     "selu",
 ]
 
-for binary_op, onnx_op in {
```

Comment:
any significance to moving it around in the file?

Reply:
no significance beyond making the file more organized. Now it is unary ops and then binary ops. before they were interwoven. Seeing it in non-diff view may help: https://github.com/microsoft/onnxruntime/blob/wilbrady/addFix/orttraining/orttraining/eager/opgen/opgen/atenops.py